applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -142,17 +167,23 @@ def _ComputeLiftFromFarField(self):
 
         for cond in self.far_field_model_part.Conditions:
             surface_normal = cond.GetGeometry().Normal()
-
-            # Computing contribution due to pressure
-            pressure_coefficient = cond.GetValue(KratosMultiphysics.PRESSURE_COEFFICIENT)
-            force_coefficient_pres -= surface_normal*pressure_coefficient
-
-            # Computing contribution due to convection
-            velocity = cond.GetValue(KratosMultiphysics.VELOCITY)
-            velocity_projection = _DotProduct(velocity, surface_normal)
-            disturbance = velocity - free_stream_velocity
-            density = cond.GetValue(KratosMultiphysics.DENSITY)
-            force_coefficient_vel -= velocity_projection * disturbance * density
+            norm = surface_normal.norm_2()
+            if abs(norm) < 1e-9:
+                raise Exception('The norm of the condition ', cond.Id , ' should be larger than 0.')
+            surface_normal_normalized = surface_normal/norm
+            span_projection = _DotProduct(surface_normal_normalized, self.span_direction)
+
+            if not self.is_infinite_wing or abs(span_projection) < 0.1:
```

Comment:
Just for my understanding, why is the span projection compared against the value 0.1? Where does it come from?

Reply:
0.1 is just a tolerance. So what I am trying to do here is select only the far field conditions (faces) whose normal doesn't have a span projection (i.e. Inlet, Outlet and Upper and Lower wall faces).